saleor/graphql/checkout/filters.py

```diff
@@ -68,6 +68,8 @@ def filter_checkout_search(qs, _, value):
 
     filter_option = Q(Exists(users.filter(id=OuterRef("user_id"))))
 
+    filter_option |= Q(metadata__icontains=value)
```

Comment:
`__icontains` is a performance killer, use `__ilike` instead.

Reply:
This is jsonb field, can't do ilike on it, AFAIK?!